lib/classes/class-gs-client.php

```diff
@@ -189,12 +190,17 @@ public function add_media( $args = array() ) {
             return new \WP_Error( 'sm_error', __( 'Unable to locate file on disk', ud_get_stateless_media()->domain ) );
           }
 
+          $use_wildcards = isset($_REQUEST['use_wildcards']) ? $_REQUEST['use_wildcards'] : false;
```

Comment:
This string appears as a duplicate many times. Probably it is better to create some utility function that will return whether to use_wildcards or not.

Reply:
Done